graphconstruction/creatingGraph.py

```diff
@@ -69,10 +69,23 @@ def prepare_dataset(rfilename, num_event, num_start=0):
     Returned is a list of graphs (torch.geometric data), with one graph for one event.
     """
     data_list = []
+    
 
     df_pf_list, df_gen_list = gen_dataframe(rfilename, num_event, num_start)
 
     PTCUT = 0.5
+    
+    #defination of edge distance
+    #deltaR < 0.8/0.3 are recognized as an edge
+    dist_phi = distance.cdist(phi, phi, 'cityblock')
```

Comment:
is this correct? why is `dist_phi` calculated before `phi` got its value from the event?